src/sasjs-context/index.js

```diff
@@ -235,6 +249,45 @@ async function edit(config, target) {
   }
 }
 
+async function remove(config, target) {
+  const sasjs = new SASjs({
+    serverUrl: target.serverUrl,
+    serverType: target.serverType
+  })
+
+  const accessToken = target.authInfo.access_token
+  const { contextId } = config
+
+  const deletedContext = await sasjs
+    .deleteContext(contextId, accessToken)
+    .catch((err) => {
+      if (err.hasOwnProperty('body')) {
+        const body = JSON.parse(err.body)
+        const message = body.message || ''
+        const details = body.details || ''
+
+        console.log(
+          chalk.redBright(
+            'An error has occurred when processing context.',
```

Comment:
Might be more useful to say `An error has occurred when deleting context ${id}.`, or even just `An error has occurred when deleting context.`

Reply:
agree, fixed